src/snippet/unminified.fetchhandler.js

```diff
@@ -0,0 +1,42 @@
+(function(wind, doc, scriptTag, url, obj, noConflict, s, n, o, f) {
```

Comment:
Asuming this script is being automatically minified, could the parameters (s, n, o, f) be given more meaningful names for greater clarity?

Reply:
Good point, I'll look at giving them some more meaningful names